### ISSUE TITLE
Link to specific releases / commits for milestone examples

### ISSUE THREAD

**Issue by chendaniely** [UBC-DSCI/dsci-310-student#51]
The current example project is in the final project state, not in the milestone state.
Students end up doing more than what they need to since they're blindly following the final project, not the instructions listed in the milestone

**Comment by tonyliang19** [comment]
Yes that's correct, examples includes and not limited to using `docker-compose.yml` in milestone 1, command line arguments for Rscripts in milestone 2. The project itself is contained in many places of the submissions for milestones like the following links:
- [Milestone 2](https://ubc-dsci.github.io/dsci-310-student/milestone2.html)
![image](https://user-images.githubusercontent.com/83138402/224872192-a3ad59aa-12c6-4e78-98f4-b12b28a9a34c.png)
The link that contains v2.0.0 isnt actually working, and the second link of the image, which is [here](https://github.com/UBC-DSCI/predict-airbnb-nightly-price), is in fact the final state of the project.
